docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
+  },
+  {
+    "id": 4,
+    "application": "application-a",
+    "profile": "production",
+    "label": "12c",
+    "propKey": "db-connection",
+    "value": "connectionstring-a-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 5,
+    "application": "application-a",
+    "profile": "development",
+    "label": "23cbeta",
+    "propKey": "db-dev-name",
+    "value": "databasename-a-dev",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 6,
+    "application": "application-a",
+    "profile": "development",
+    "label": "23cbeta",
+    "propKey": "db-dev-connection",
+    "value": "connectionstring-a-dev",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>
+
+Get all properties for a service-name (application) filtered on service-label (label).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-b& \
+  service-label=19c
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "id": 7,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-name",
+    "value": "databasename-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 8,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-connection",
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile) and service-profile (profile).
+
+```shell
+curl -s http://localhost:8080/srv/config/properties? \
+  service-name=application-b& \
+  service-label=19c& \
+  service-profile=production
+```
+
+Example of data returned:
+
+```json
+[
+  {
+    "id": 7,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-name",
+    "value": "databasename-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  },
+  {
+    "id": 8,
+    "application": "application-b",
+    "profile": "production",
+    "label": "19c",
+    "propKey": "db-connection",
+    "value": "connectionstring-b-prod",
+    "createdOn": "2023-10-19T16:50:07.000+00:00",
+    "createdBy": "ADMIN"
+  }
+]
+```
+
+#### /srv/config/properties?service-name=\<service-name\>&service-label=\<service-label\>&service-profile=\<service-profile\>&property-key=\<property-key\>
+
+Get all properties for a service-name (application) filtered on service-profile (profile), service-profile (profile) and property-key (prop_key).
```

Comment:
Change the period to a colon.

You have used "service-profile (profile)" twice in this sentence. Is that intended?

Add a comma before "and".